website/docs/d/healthcare_service.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_healthcare_service"
+sidebar_current: "docs-azurerm-datasource-healthcare-service-x"
+description: |-
+  Get information about an existing Healthcare Service
+
+---
+
+# Data Source: azurerm_healthcare_service
+
+Use this data source to access information about an existing Healthcare Service
+
+## Example Usage
+
+```hcl
+data "azurerm_healthcare_service" "test" {
+  name                = "example-healthcare_service"
+  resource_group_name = "example-resources"
+  location            = "westus2"
+}
+
+output "healthcare_service_id" {
+    name                = "uniquefhirname"
+    resource_group_name = "sample-resource-group"
+    kind = "fhir-R4"
```

Comment:
Could we properly align these assingments?

Reply:
done

Reply:
done